### PR TITLE
Rubocop configuration

### PR DESCRIPTION
**Pull request by FranDepascuali**
## Sumary

Added rubocop to project.

**Trello cards**:
https://trello.com/c/iIed6xIZ/9-configurar-rubocop
## Test Plan
- Open a terminal.
- Go to project directory.
- Run rubocop.
## Screenshots

rubocop

### CODE REVIEW

**Comment by FranDepascuali** [comment]
@NicoBuch

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
+inherit_from: .rubocop-todo.yml
```
Comment (NicoBuch) [inline .rubocop_todo.yml]: @FranDepascuali Check out this file: You have everything disabled. 
This file should only have the AllCops and 

``` ruby
Documentation:
  Enabled: false
LineLength:
  Max: 99
```

**Comment by FranDepascuali** [comment]
@NicoBuch

File: .rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+AllCops:
+ Exclude:
+   - features/support/env.rb
+   - spec/spec_helper.rb
+   - features/step_definitions/email_steps.rb
+   - spec/controllers/v1/users_controller_spec.rb
```
Comment (NicoBuch) [inline .rubocop.yml]: Do this files exist in the proyect? If not, delete this lines :)

**Comment by FranDepascuali** [comment]
@NicoBuch

**Comment by FranDepascuali** [comment]
@NicoBuch

**Comment by NicoBuch** [comment]
:+1:

**Pull request merged**